.github/workflows/cicd.yml

```diff
@@ -16,14 +16,14 @@ jobs:
     if: github.repository != 'Exabyte-io/template-definitions'
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [12.x, 14.x, 16.x, 18.x]
```

Comment:
We can drop 12 here and elsewhere where we touch similar files in other packages